aten/src/ATen/detail/XPUHooksInterface.h

```diff
@@ -39,7 +39,7 @@ struct TORCH_API XPUHooksInterface {
         XPU_HELP);
   }
 
-  virtual int getGlobalIdxFromDevice(const Device& device) const {
+  virtual DeviceIndex getGlobalIdxFromDevice(const Device& device) const {
```

Comment:
```suggestion
  virtual int32_t getGlobalIdxFromDevice(const Device& device) const {
```

Reply:
This API in hooks is used in dlpack.

Reply:
dlpack was changed to this new API in the PR